scalastyle/scalastyle-checks.xml

```diff
@@ -61,7 +61,7 @@
  <check level="warning" class="org.scalastyle.scalariform.EqualsHashCodeChecker" enabled="true"></check>
  <check level="warning" class="org.scalastyle.scalariform.IllegalImportsChecker" enabled="true">
   <parameters>
-   <parameter name="illegalImports"><![CDATA[sun._,java.awt._]]></parameter>
+   <parameter name="illegalImports"><![CDATA[sun._]]></parameter>
```

Comment:
Is it known why importing java.awt classes is encouraged by the original authors of this file?

Reply:
No, I don't know why java.awt is considered "illegal". But I was able to find such rule in Scalastyle configurations dating as back as 2012, so I don't think anybody remembers the true reason.

Anyway, I consider this wrong, so I changed it :p